### DISCUSSION TITLE
Computation inside markdown text: is there some documentation?

### DISCUSSION THREAD

**Discussion by aborruso** [quarto-dev/quarto-cli#3606] [Q&A] [answered]
Hi,
in this [good presentation](https://youtu.be/shVSmYna3GM?t=1959)  of R-Ladies, Julia Muller uses a qmd file and this markdown text:

```
The average bill length is `r mean(penguins$bill_length_mm)` mm and the average bill depth is `r mean(penguins$bill_depth_mm)` mm.
```

The computation takes place within the markdown text and not inside a code block. 

I have two (probably) stupid question:

- where do I find some notes about this, inside quarto documentation?
- could I do the same for all quarto usable kernels (python, julia, ecc.)?

Thank you

**Comment by cscheid** [comment, marked as answer]
That's a knitr feature. We don't advertise it broadly in quarto because Jupyter doesn't have a corresponding feature.

The recommended workaround is to use `Display(Markdown())` in Jupyter.

**Reply by aborruso** [reply]
For an app as Quarto, in my opinion, it would be great to have a general way to mix markdown and inline code.

**Reply by cscheid** [reply]
`Display(Markdown())` is general, though I agree it's not ergonomic. What we don't want to do is come up with a solution that doesn't play well with our Jupyter compatibility requirements. We need to be able to import and export from and to `.ipynb`, and inline execution in plain markdown doesn't currently play nicely with that.

**Reply by aborruso** [reply]
thank you @cscheid
